product.py

```diff
@@ -46,12 +84,21 @@ class ProductTemplate:
         'get_products_displayed_on_eshop'
     )
     description = fields.Text("Description")
-    image_sets = fields.One2Many(
-        'product.product.imageset', 'template', 'Images',
-    )
-    default_image_set = fields.Many2One(
-        'product.product.imageset', 'Default Image Set', readonly=True
+    static_files = fields.One2Many(
```

Comment:
I think you should add an `add_remove` clause to add existing static files that may already be there. 
Giving another thought I am wondering why not have this as a `many2many` field ?